Possible TCK bug - reporting failure when using `OR` logic - basicSchemaTests 49

While working on the SmallRye impl for base CDI/reflection, I ran into this TCK test failure:
```
[ERROR]   Run 115: SchemaDynamicValidityTest>Arquillian.run:138->testPartsOfSchema:87 [Basic Scalar Types] (49) - Expecting a Mutation with the set removed from the name, scalarHolder. expected [true] but found [false]
```

Test 49 looks like this in the CSV file:
```
49| type Mutation           |   scalarHolder(arg0: ScalarHolderInput): ScalarHolder 'OR' scalarHolder(scalarHolder: ScalarHolderInput): ScalarHolder   |   Expecting a Mutation with the set removed from the name, scalarHolder.
```

However, the schema generated by my implementation produces the correct text that is before the "'OR'" text:
```
  scalarHolder(arg0: ScalarHolderInput): ScalarHolder
```

I think that there might be an issue with the 'OR' logic.

--- a/tck/src/main/java/org/eclipse/microprofile/graphql/tck/dynamic/schema/SchemaTestDataProvider.java
+++ b/tck/src/main/java/org/eclipse/microprofile/graphql/tck/dynamic/schema/SchemaTestDataProvider.java
@@ -132,7 +132,7 @@ public class SchemaTestDataProvider {
         if(containsString.contains(OR)){
             String[] containsStrings = containsString.split(OR);
             for(String oneOf:containsStrings){
-                testData.addContainsString(oneOf);
+                testData.addContainsString(oneOf.trim());
             }
         }else{
             testData.addContainsString(containsString);

